src/System.Management.Automation/engine/parser/SafeValues.cs

```diff
@@ -609,6 +648,16 @@ public object VisitPipeline(PipelineAst pipelineAst)
             throw PSTraceSource.NewArgumentException("ast");
         }
 
+        public object VisitTernaryExpression(TernaryExpressionAst ternaryExpressionAst)
+        {
+            if (t_context == null)
+            {
+                throw PSTraceSource.NewArgumentException("ast");
```

Comment:
Is this the right error to throw here? Seems like it would be misleading

Reply:
Hmm, not the right error. Will update them all.